csunplugged/tests/resources/views/test_index_view.py

```diff
@@ -38,7 +38,7 @@ def test_resources_index_with_multiple_resources(self):
             "binary-cards",
             "Binary Cards",
             "resources/binary-cards.html",
-            "binary_cards.py",
+            "BinaryCardsResourceGenerator",
         )
         self.test_data.create_resource(
```

Comment:
`sorting_network.py` doesn't exist anymore

Reply:
this test passes despite `sorting_network.py` not existing anymore, does that seem weird do you? I understand why it passes, but it seems wrong that it should.

Reply:
This is because the resource is added with the test loader instead of the resource loader, I've created issue #639 to investigate this.